apps/snfoundry/contracts/src/cofi_collection.cairo

```diff
@@ -179,35 +174,31 @@ pub trait ICofiCollection<TContractState> {
 
 #[starknet::contract]
 mod CofiCollection {
-    use openzeppelin::{
-        access::accesscontrol::{AccessControlComponent, DEFAULT_ADMIN_ROLE},
-        introspection::src5::SRC5Component, security::pausable::PausableComponent,
-        token::erc1155::ERC1155Component, upgrades::{UpgradeableComponent, interface::IUpgradeable}
-    };
+    use core::num::traits::Zero;
+    use openzeppelin::access::ownable::OwnableComponent;
```

Comment:
Changing auth model of this contract to ownership model... only marketplace will have ownership of it, so we don't need roles in here.